pkg/services/ngalert/api/fork_ruler.go

```diff
@@ -2,26 +2,57 @@ package api
 
 import (
 	"fmt"
+	"time"
 
 	"github.com/grafana/grafana/pkg/api/response"
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/services/datasources"
 	apimodels "github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	GrafanaBackend = "grafana"
+	ProxyBackend   = "proxy" // metric cardinality is too high to enumerate all non-grafana backends
```

Comment:
Wouldn't it be at most 3 values? Loki, Cortex and Grafana? I'm probably missing something, ain't I?

Reply:
Oof, good catch. I believe my intiial thought was that we opened ourselves the opportunities to add more "managed alerting backends", but no reason to preemptively organize around that problem. Will adjust.